### PR TITLE
Support sun JMS Clients

### PR DESCRIPTION
**Pull request by realark**
* Allow instrumenting com.sun.messaging. namespace

https://github.com/DataDog/dd-trace-java/issues/517

### CODE REVIEW

**Comment by realark** [comment]
@tony-schellenberg Would you mind testing out a snapshot build with this fix? It's the 0.16.0 release plus this change: https://10065-89221572-gh.circle-artifacts.com/0/home/circleci/dd-trace-java/libs/dd-java-agent-0.17.0-SNAPSHOT.jar

**Comment by tony-schellenberg** [comment]
I had a quick look at it and it seems to be instrumenting our JMS now. I'm not highly familiar with the particular service so I'm not sure it's 100% correct but I can see spans showing up now.

Thanks for helping out on this.

File: dd-java-agent/agent-tooling/src/main/java/datadog/trace/agent/tooling/AgentInstaller.java [hunk 1]
```diff
@@ -135,7 +135,9 @@ public void onIgnored(
         final TypeDescription typeDescription,
         final ClassLoader classLoader,
         final JavaModule module,
-        final boolean loaded) {}
+        final boolean loaded) {
+      log.debug("Ignored {} -- {}", typeDescription, classLoader);
```
Comment (tylerbenson) [inline dd-java-agent/agent-tooling/src/main/java/datadog/trace/agent/tooling/AgentInstaller.java]: How much log noise does this generate when enabled?

Comment (tony-schellenberg) [reply dd-java-agent/agent-tooling/src/main/java/datadog/trace/agent/tooling/AgentInstaller.java]: Not sure if you were asking @realark or me. I just ran it on a relatively small application we have and it generated a lot of logging.

In this case, which was a small test run to validate this, it generated 6310 log lines (93% of the log lines were this, total 6749 log lines).

Comment (realark) [reply dd-java-agent/agent-tooling/src/main/java/datadog/trace/agent/tooling/AgentInstaller.java]: Ouch. Probably best to leave this out then. Thanks for the input.

Comment (tylerbenson) [reply dd-java-agent/agent-tooling/src/main/java/datadog/trace/agent/tooling/AgentInstaller.java]: I was asking @realark but that was valuable input. Thanks @tony-schellenberg.

**Pull request merged**